Tests/test-dataframe.lisp

```diff
@@ -271,6 +271,71 @@
                                  (funcall gen-index*))
                            (return array))))))))
 
+
+
```

Comment:
please remove unnecessary empty lines (one empty line between toplevel forms is plenty, if you feel that things do not belong to the same page insert a break page character -- in emacs C-q C-l)

Reply:
Done.